wandb/sdk/wandb_run.py

```diff
@@ -930,7 +930,7 @@ def _history_callback(self, row: Dict[str, Any], step: int) -> None:
         if self._backend:
             not_using_tensorboard = len(wandb.patched["tensorboard"]) == 0
             self._backend.interface.publish_history(
-                row, step, publish_step=not_using_tensorboard
+                row, step, run=self, publish_step=not_using_tensorboard
```

Comment:
Whoa, why was this needed?

Reply:
This was a relic of previous experimentation, removed.